bin/logagent-setup.sh

```diff
@@ -20,6 +20,36 @@ while getopts ":i:u:g:" opt; do
     i)
       export TOKEN=$OPTARG
       ;;
+    j)
+      export JOURNALD=yes
+      if [ ! -f /lib/systemd/system/systemd-journal-upload.service ]; then
+        echo
+        echo "journal-upload service not found. Attempting to install it..."
+        echo
+        command -v apt-get > /dev/null
```

Comment:
You can simplify the command check with:

```
if [ -x "$(command -v apt-get)" ]; then
```

Reply:
Done. Thanks!